### ISSUE TITLE
yb-master: compact_table command fails even though the underlying compaction actually succeeded.

### ISSUE THREAD

**Issue by rthallamko3** [yugabyte/yugabyte-db#10213]
Jira Link: [DB-1022](https://yugabyte.atlassian.net/browse/DB-1022)
Mac hostname resolution causes a bunch of spam to master logs. Once the resolution is done, all of that is sent to the raft metadata which overloads master too. 

Other interesting side effects - manual command to compact fails.

Raghavendras-MacBook-Pro:bin tk$ ./yb-admin compact_table ybdemo_keyspace cassandrakeyvalue 300
Error running compact_table: Timed out (yb/client/client-internal.cc:144): Unable to compact table ybdemo_keyspace.cassandrakeyvalue: Timed out waiting for FlushTables

[yb-master.INFO.zip](https://github.com/yugabyte/yugabyte-db/files/7297877/yb-master.INFO.zip)

**Comment by rthallamko3** [comment]
With the recent work on making compaction command async, this is no longer needed.
